### PR TITLE
feat(sca): update runtime page to take into account sca

### PR DESCRIPTION
**Pull request by danila-m**
Depends on https://github.com/bonitasoft/bonita-doc/pull/2455

### CODE REVIEW

File: modules/runtime/pages/bonita-applications-interface-overview.adoc [hunk 1]
```diff
@@ -22,9 +22,9 @@ Bonita Runtime embeds two types of applications:
   ** _User_ , based on a standard BPM usage, centered on a generic tasklist
   ** _Administrator_, allowing the management (monitoring and troubleshooting) of an automation project in production (BDM excepted)
 
-These applications are installed by default for a fresh installation of the platform. +
-In addition to that, the generic Bonita Applications can be downloaded from the Welcome Page of the Studio, in the "Resources" tile. +
-You may want to do so to create your applications from an example. +
+Core applications are installed by default and cannot be removed.
+Generic applications will be installed on a fresh installation. They can also be downloaded from the Welcome Page of the Studio, in the "Resources" tile. +
+You may want to do so to create your applications from an example, or to include these applications in your self-contained bonita application (see: xref::application-build.adoc)
```
Comment (jeromecambon) [inline modules/runtime/pages/bonita-applications-interface-overview.adoc]: Invalid xref. 
Should be xref:runtime:application-build.adoc
See https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#adding-reference-between-pages-of-two-components-or-between-pages-of-2-versions-of-the-same-component

File: modules/runtime/pages/custom-deployment.adoc [hunk 2]
```diff
@@ -6,8 +6,8 @@
 
 [WARNING]
 ====
-Remember that the recommended way of installing Bonita is to use the provided xref:tomcat-bundle.adoc[Tomcat] bundle. +
-It saves you from doing all the following configuration at hand, as the xref:runtime:bonita-platform-setup.adoc[setup tool] included handles it automatically.
+Remember that the recommended way of installing Bonita is through a self-contained application xref:application-build.adoc
```
Comment (jeromecambon) [inline modules/runtime/pages/custom-deployment.adoc]: Invalid xref. 
Should be xref:runtime:application-build.adoc
See https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#adding-reference-between-pages-of-two-components-or-between-pages-of-2-versions-of-the-same-component

Comment (benjaminParisel) [reply modules/runtime/pages/custom-deployment.adoc]: done :heavy_check_mark: :

File: modules/runtime/pages/bonita-applications-interface-overview.adoc (hunk 1 above)
Comment (benjaminParisel) [reply modules/runtime/pages/bonita-applications-interface-overview.adoc]: done :heavy_check_mark:

**Pull request merged**